### PR TITLE
#164798247 Setup React Router version 4

### PR DESCRIPTION
**Pull request by patrickf949**
#### What does this PR do?
Sets up react router.

#### Description of tasks to be done.
- Configure and install react router
- create dummy route for login and landing page

#### How should this be manually tested?
Clone the repo and check into its directory
Install dependencies

- `$ npm install`

Start the server

- `$ npm start`

In your browser you should be able to see the landing page with this link

```localhost:8080```

Try out this URL to view the dummy login page

```localhost:8080/login```

#### Screenshots

Landing Page

Screenshot 2019-05-01 at 13 46 37

Dummy Login Page

Screenshot 2019-05-01 at 13 46 53

#### Relevant Pivotal tracker stories

- [#164798247](https://www.pivotaltracker.com/story/show/164798247)

### CODE REVIEW

**Review by e-ian — COMMENTED** [review]
@patrickf949 kindly update screenshots in PR to show styling with scss

**Review by anyatibrian — APPROVED** [review]
LGTM

**Comment by patrickf949** [comment]
> @patrickf949 kindly update screenshots in PR to show styling with scss

Thanks @e-ian, All done

**Review by Genza999 — APPROVED** [review]
LGTM

File: src/components/home.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```
Comment (MuhanguziDavid) [inline src/components/home.js]: Use indentation of two(2) spaces for this file and any other file

**Pull request merged**